file/src/od.rs

```diff
@@ -0,0 +1,962 @@
+use crate::io::ErrorKind;
```

Comment:
Please add a standard copyright header to all .rs source files - copyright Hemi Labs etc.